kit/src/main/java/com/oracle/javafx/scenebuilder/kit/util/PaintConvertUtil.java

```diff
@@ -0,0 +1,145 @@
+package com.oracle.javafx.scenebuilder.kit.util;
+
+import javafx.scene.paint.*;
+
+import java.util.List;
+
+/**
+ * Convert Paint to java code or css code;
+ */
+public class PaintConvertUtil {
+
+    private static final int ROUNDING_FACTOR = 10000;//Use for round to 4 decimal places
+
+    public static String convertPaintToCss(Paint fxPaint) {
```

Comment:
Please add some tests to confirm the validity of this method.

Reply:
Thanks, the test class has been added;
com.oracle.javafx.scenebuilder.kit.util.PaintConvertUtilTest